index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header-box">
+        <p>John Doe</p>
+        <img src="/images/Menu-icon.png" alt="" />
```

Comment:
- [ ] Please consider wrapping the menu-icon with a `nav` tag as the menu-icon which is known as the hamburger menu consists of navigation links and also to improve the semantic structure of your HTML😄. You can read more on [SEMANTIC HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). 
- [ ] Also, please add the `alt` text of the menu icon so as to improve accessibility and to help visually impaired people to access your page. See my suggestions below.

```suggestion
     <nav>
        <img src="/images/Menu-icon.png" alt="Menu icon" />
     </nav>
```